packages/web/components/table/asset-info.tsx

```diff
@@ -0,0 +1,669 @@
+    </div>
+  );
+});
+
+type AssetInfoCellComponent<TProps = {}> = FunctionComponent<
+  CellContext<AssetInfo, AssetInfo> & TProps
+>;
+
+const AssetCell: AssetInfoCellComponent<{
+  isFavorite: boolean;
+  onSetFavorite: () => void;
+  onRemoveFavorite: () => void;
+}> = ({
+  row: {
+    original: { coinDenom, coinName, coinImageUrl, isVerified },
+  },
+  isFavorite,
+  onSetFavorite,
+  onRemoveFavorite,
+}) => {
+  const [isHover, setHover] = useState(false);
+
+  return (
+    <div
+      className={classNames("flex items-center gap-2", {
+        "opacity-40": !isVerified,
+      })}
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+    >
+      {isFavorite || isHover ? (
+        <div className="cursor-pointer">
+          <Icon
+            id="star"
+            className={classNames(
+              isHover ? "text-osmoverse-600" : "text-wosmongton-400"
+            )}
+            onClick={(event) => {
+              event.preventDefault();
+              event.stopPropagation();
+
+              if (isFavorite) onRemoveFavorite();
+              else onSetFavorite();
+            }}
+            height={24}
+            width={24}
+          />
+        </div>
+      ) : (
+        <div style={{ height: 24, width: 24 }} />
+      )}
+      <div className="flex items-center gap-4">
+        <div className="h-10 w-10">
+          {coinImageUrl && (
+            <Image alt={coinDenom} src={coinImageUrl} height={40} width={40} />
+          )}
+        </div>
+        <div className="subtitle1 flex max-w-[200px] flex-col place-content-center">
+          <div className="flex">
+            <span className="text-white-high">{coinDenom}</span>
+          </div>
+          <span className="overflow-hidden overflow-ellipsis whitespace-nowrap text-osmoverse-400">
+            {coinName}
+          </span>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const PriceCell: AssetInfoCellComponent = ({
+  row: {
+    original: { currentPrice, priceChange24h },
+  },
+}) => (
+  <div className="flex flex-col">
+    {currentPrice && (
+      <span className="subtitle1">{currentPrice?.toString()}</span>
+    )}
+    {priceChange24h && (
+      <span
+        className={classNames("caption", {
+          "text-bullish-400":
+            priceChange24h && priceChange24h.toDec().isPositive(),
+          "text-ammelia-400":
+            priceChange24h && priceChange24h.toDec().isNegative(),
+        })}
+      >
+        {priceChange24h && priceChange24h.toDec().isPositive() ? "+" : null}
+        {priceChange24h.maxDecimals(1).toString()}
+      </span>
+    )}
+  </div>
+);
+
+const SparklineChartCell: AssetInfoCellComponent<{
+  timeFrame: CommonHistoricalPriceTimeFrame;
+}> = ({
+  row: {
+    original: { coinDenom, priceChange24h },
+  },
+  timeFrame,
+}) => {
+  const { data: recentPrices } =
+    api.edge.assets.getAssetHistoricalPrice.useQuery({
+      coinDenom,
+      timeFrame,
+    });
```

Comment:
What are your thoughts on adding a stale time of 30 seconds to avoid fetching constantly while user scrolls